addons/binding/org.openhab.binding.xmppclient/src/main/java/org/openhab/binding/xmppclient/handler/XMPPClientConfiguration.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.xmppclient.handler;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link XMPPClientConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Pavel Gololobov - Initial contribution
+ */
+@NonNullByDefault
+public class XMPPClientConfiguration {
+
+    public String host;
```

Comment:
Does that compile? `NonNullByDefault` usually forces you to assign default values or annotate your fields with `@Nullable`.